### ISSUE TITLE
Feature: add a lego-table interface

### ISSUE THREAD

**Issue by MathiasBaumgartinger** [boku-ilen/landscapelab#331]
Similarly to the UI provided in https://github.com/boku-ilen/landscapelab-table

**Comment by kb173** [comment]
First prototype of the data sent by the table:

```
{"event": "brick_added", "data": {"id": 5, "position": [0.7296875, 0.38472222222222224], "shape": "BrickShape.SQUARE_BRICK", "color": "BrickColor.RED_BRICK"}}
{"event": "brick_removed", "data": {"id": 5}}
```

Any objections?

This is probably all the table will ever need to communicate, so the generic separation into "event" and "data" is probably redundant, but I think it seems more well-structured this way

**Comment by MathiasBaumgartinger** [comment]
Position will always be an array between 0 and 1 for both x and y axis, right? 

I also agree. A clear distinction between the data and the actual event is preferable, as it is more flexible/modular. No objections.

**Comment by kb173** [comment]
Yes, position is always [0..1, 0..1]. cool, the adaptations to the table code should be finished then!

**Comment by kb173** [comment]
I did some testing with MBTiles so we can load nice maps per zoom level. MBTiles only support WebMercator, so our LabTable map will need to use a different projection than the rest of the LandscapeLab. This should not be a big problem because Geodot provides `GeoTransform.transform_coordinates`. For example:

```
print("Center in EPSG 32633:", GeoTransform.transform_coordinates(
    Vector3(start_position_x, 0.0, start_position_y), 3857, 32633
))
```

Loading MBTiles at specific zoom levels works out of the box, we just need to enter the correct resolution. This code snippet loads pixel-perfect maps by zoom level and resolution:

```
const highest_zoom := 19
const resolution_at_highest_zoom := 0.3

var zoom_level = 13
var tile_size_pixels := 256.0

var tile_size_meters = tile_size_pixels * \
		(resolution_at_highest_zoom * \
        pow(2, highest_zoom + 1 - zoom_level))
```

**Comment by MathiasBaumgartinger** [comment]
Nice! Looks promising. If you want to try - https://github.com/boku-ilen/landscapelab/tree/lego-table has the first draft for the table interface. 

For debugging purposes, i designed it such that you can start `UI/LegoTable/LegoTable.tscn` standalone. It will just load the `config.ll` as always but without all the 3D configuration overhead.

**Comment by MathiasBaumgartinger** [comment]
Branch https://github.com/boku-ilen/landscapelab/tree/lab-table implements features required
